drivers/volume/portworx/portworx.go

```diff
@@ -2156,7 +2152,9 @@ func (d *portworx) CollectDiags(n node.Node) error {
 		Sudo:            true,
 	}
 
-	logrus.Debugf("Collecting diags on node %v, because there was an error", pxNode.Hostname)
+	if !diagOps.Validate {
+		logrus.Infof("Collecting diags on node %v, because there was an error", pxNode.Hostname)
```

Comment:
From an interface implementation standpoint, it should not assume why someone invoked it. You can simply say

`Collecting diags on node %v. Will skip validation.`